frontend/src/metabase/nav/components/DatabasePromptBanner/DatabasePromptBanner.tsx

```diff
@@ -42,9 +43,21 @@ export function DatabasePromptBanner({ location }: DatabasePromptBannerProps) {
     <DatabasePromptBannerRoot role="banner">
       <Prompt>{t`Connect to your database to get the most from Metabase.`}</Prompt>
       <CallToActions>
-        <GetHelpButton href="https://metabase.com/help/connect">{t`Get help connecting`}</GetHelpButton>
+        <GetHelpButton
+          href="https://metabase.com/help/connect"
+          onClickCapture={e => {
+            // We do this in ExternalLink
+            e.stopPropagation();
```

Comment:
I need to repeat what we've done in `ExternalLink` here. Since we stop the propagation there. I can't simply pass another `onClick` handler since the capture event handlers will run before the bubbling event handlers.

https://github.com/metabase/metabase/blob/049608de5b112f90f262dc57704dc93a0513efed/frontend/src/metabase/core/components/ExternalLink/ExternalLink.tsx#L25

Reply:
I think there is no harm in letting the event propagate here, no?

Reply:
I believe you're right. Here's the full code (should have quoted the comment too in the first place)

https://github.com/metabase/metabase/blob/7a23d260184c8dc31ecd44291dd52770ceb8734a/frontend/src/metabase/core/components/ExternalLink/ExternalLink.tsx#L24-L25

so, it does that when it's in a table, but we don't need that in this case. I can remove it.

Reply:
Removed